OpenRA.Mods.Common/InstallUtils.cs

```diff
@@ -39,6 +39,12 @@ public class ContentInstaller : IGlobalModData
 		public readonly string MusicPackageMirrorList = null;
 		public readonly int ShippedSoundtracks = 0;
 
+		// InstallShield .cab File Ids, used to extract Mod specific files
+		public readonly int[] IscfFileIds = { };
+
+		// InstallShield .cab File Ids, used to extract Mod specific archives and extract contents of ExtractFilesFromCD
+		public readonly string[] IscfPackageIds = { };
```

Comment:
This is confusing. I keep reading it as "Is CF Package IDs", which suggests a `bool` value. How about renaming this (and the one above) to `InstallShieldCabFilePackageIds`? Don't be afraid of long names. That would make it much clearer what this is when looking at a `mod.yaml` file, when the person doesn't have prior knowledge.